cdap-watchdog/src/test/java/co/cask/cdap/logging/appender/TestResilientLogging.java

```diff
@@ -105,7 +106,7 @@ public void testResilientLogging() throws Exception {
       new ZKClientModule(),
       new KafkaClientModule(),
       new DiscoveryRuntimeModule().getInMemoryModules(),
-      new LocationRuntimeModule().getInMemoryModules(),
+      new LocationUnitTestModule().getModule(),
```

Comment:
Would be a good idea to rename `LocationUnitTestModule` to indicate its functionality.